### PR TITLE
Ira/first

### PR DESCRIPTION
**Pull request by irockk**

### CODE REVIEW

File: lib/screens/MainScreen.dart [hunk 1]
```diff
@@ -20,96 +20,226 @@ class SizeConfig {
...
+                        ),
+                      ),
+                      SizedBox(width: SizeConfig.blockSizeVertical * 1.6),
+                      new SimpleButton(
+                        width: SizeConfig.blockSizeVertical * 10.8,
+                        height: SizeConfig.blockSizeVertical * 10.8,
+                        margin: EdgeInsets.symmetric(
+                            vertical: SizeConfig.blockSizeVertical * 1.6),
+                        child: Text(
+                          '3',
```
Comment (chummy-ch) [inline lib/screens/MainScreen.dart]: Its code duplication. If you use same widgets with the same params you can extract them to separete class and reuse them.

File: lib/screens/MainScreen.dart [hunk 2]
```diff
@@ -20,96 +20,226 @@ class SizeConfig {
...
+                          style: TEXT_THEME.headline1,
+                        ),
+                      ),
+                      SizedBox(width: SizeConfig.blockSizeVertical * 1.6),
```
Comment (chummy-ch) [inline lib/screens/MainScreen.dart:116]: It is better to create a const value instead of using a magic integer

**Review by chummy-ch — CHANGES_REQUESTED** [review]
Should make some changes

**Comment by RuslanProgrammer** [comment]
Oh, no. You maybe forgot to merge develop branch into your firstly!?

**Review by chummy-ch — APPROVED** [review]

**Pull request merged**